scripts/ci/check_changeset.ts

```diff
@@ -28,6 +29,14 @@ const git = simpleGit(root);
 const baseRef = process.env.GITHUB_PULL_REQUEST_BASE_SHA || 'master';
 const headRef = process.env.GITHUB_PULL_REQUEST_HEAD_SHA || 'HEAD';
 
+const githubOutputFile = (function (): string {
+  const value = process.env.GITHUB_OUTPUT;
+  if (!value) {
+    throw new Error('GITHUB_OUTPUT environment variable must be set');
```

Comment:
Out of curiosity, what was the case where GITHUB_OUTPUT was not set? Is this for local testing? With this error does it get far enough in local testing for the output to be useful?

Reply:
I never encountered that. I just added that check so that the TypeScript compiler would type `githubOutputFile` as `string` instead of `string | undefined`.

Reply:
The later call to `writeFile(githubOutputFile, ...)` will fail to compile if `githubOutputFile` is `string | undefined`; however, it compiles fine if it is `string`.